iina/ja.lproj/PrefUIViewController.strings

```diff
@@ -103,6 +103,9 @@
 /* Class = "NSButtonCell"; title = "Always float on top while playing"; ObjectID = "SFX-ct-8MB"; */
 "SFX-ct-8MB.title" = "再生中はウィンドウを最前面に固定";
 
+/* Class = "NSButtonCell"; title = "Always show float on top status in the title bar"; ObjectID = "ih8-5a-PYY"; */
+"ih8-5a-PYY.title" = "見出し欄には常にトップに浮いた状態が表示される";
```

Comment:
I'm not a maintainer or anything, but this translation is just awful. This will literally mean "The state of being floated always on top are shown in the (news) headline section". The term for "top" is also just transliterated from English and is not a native word.
I would suggest leaving all other languages (except Chinese I guess) the same as the English and let other contributors translate it through Crowdin.

Reply:
Sorry, I am not a professional. 
Thank you for the changes！